dags/imagegen_devx/maxdiffusion_e2e.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A DAG to run end-to-end MaxText JAX Stable Stack tests."""
+
+
+import datetime
+from airflow import models
+from dags import composer_env, test_owner, gcs_bucket
+from dags.vm_resource import Project, TpuVersion, CpuVersion, Zone, DockerImage, GpuVersion, ClusterName
+from dags.imagegen_devx.configs import gke_config as config
+from xlml.utils import name_format
+
+# Run once a day at 4 am UTC (8 pm PST)
+SCHEDULED_TIME = "0 4 * * *" if composer_env.is_prod_env() else None
```

Comment:
Nit: ensure you have manually checked existing runs to avoid traffic for scheduling.

Reply:
@RissyRan I didn't quite catch that. Can you please elaborate on it ?

Reply:
You could check this at glance on [UI](https://screenshot.googleplex.com/6t658C5npds23w2). So, we won't put all dag runs at 4am UTC ;)